src/@types/event.ts

```diff
@@ -144,6 +144,13 @@ export const UNSTABLE_MSC3089_LEAF = new UnstableValue("m.leaf", "org.matrix.msc
  */
 export const UNSTABLE_MSC3089_BRANCH = new UnstableValue("m.branch", "org.matrix.msc3089.branch");
 
+/**
+ * Functional members type for declaring a purpose of room members (e.g. helpful bots).
+ * Note that this reference is UNSTABLE and subject to breaking changes, including its
+ * eventual removal.
+ */
+export const UNSTABLE_ELEMENT_FUNCTIONAL_USERS = new UnstableValue("io.element.functional_members");
```

Comment:
@turt2live Can I give it the same value as the first and second parameter?

Reply:
In general, how to use this `UnstableValue`?
TypeScript complains.
https://buildkite.com/matrix-dot-org/matrix-js-sdk/builds/3082#5440f81c-35a8-44de-a76d-7013d2f8b207